src/GridFS/Bucket.php

```diff
@@ -381,6 +400,10 @@ public function getFileDocumentForStream($stream)
     {
         $file = $this->getRawFileDocumentForStream($stream);
 
+        if ($this->codec instanceof DocumentCodec) {
```

Comment:
Why is there this specific codec test here?

Reply:
No particular reason, I've shortened it.